src/include/distributed/metadata_utility.h

```diff
@@ -177,6 +177,16 @@ typedef struct TableConversionReturn
 }TableConversionReturn;
 
 
+/* Size queries for PG and Citus*/
+typedef enum SizeQueryType
+{
+	RELATION_SIZE,
+	TOTAL_RELATION_SIZE,
+	TABLE_SIZE,
```

Comment:
what is the difference between a `RELATION_SIZE` and a `TABLE_SIZE`.
I do see that the original code called `pg_table_size` is the result of that function different compared to `pg_relation_size` ?

A comment on these values describing their subtile differences would be beneficial.